src/Routes/RoutesCommon.tsx

```diff
@@ -218,7 +218,7 @@ export const routes: Route[] = [
     priority: 7,
   },
   {
-    path: '/rfox',
+    path: '/rfox/*',
```

Comment:
Routing rework was needed to make the claim click work 
https://jam.dev/c/3d0251f0-777b-4a07-99d3-f27da3ac0c40

This now goes from being `/#/rfox` and fully `<MemoryRouter />`s across the board to having top-level routes for tabs, as well as having sub-routes handled for claims (the rest stays as it was, except for the tab browser routes addition)